x/mongo/driver/operation.go

```diff
@@ -562,7 +559,7 @@ func (op *Operation) moreToComeRoundTrip(ctx context.Context, conn Connection, w
 
 // decompressWireMessage handles decompressing a wiremessage. If the wiremessage
 // is not compressed, this method will return the wiremessage.
-func (Operation) decompressWireMessage(wm []byte) ([]byte, error) {
+func decompressWireMessage(wm []byte) ([]byte, error) {
```

Comment:
Is there a particular reason you removed this method from the `Operation` type? I think this function would only be used by the `Operation` type.

Reply:
I made it static since it didn't need the type, but this has been reverted